src/classwork/04_assign/main.cpp

```diff
@@ -10,6 +12,46 @@ factorial.  Also, loop continues as long as user wants to.
 */
 int main() 
 {
+	//Hopefully this works, I put the loop true right now so that it wont be infinite.
+	int msg_prompt = 1;
+
+	//Loop
+	while(msg_prompt == 1)
+	{
+		//variables
+		int keyboard_number;
+		cout<<"enter a number from 1 to 10: ";
+		cin>>keyboard_number;
+
+		//If number entered is a negative number
+		if(keyboard_number < 0)
+		{
+			cout<<"\nwe cant accumulate negative numbers"<<"\npress 1 to try again or press any button to quit: ";
+			//Will ask the user if they want to stop the loop
+			cin>>msg_prompt;
+
+		}
+		
+		// If number entered is in range of 1 to 10
+		else if (keyboard_number >= 1 && keyboard_number <= 10)
+		{
+			cout<<"\nYour factorial number is: "<<factorial(keyboard_number)<<"\npress 1 to try again or press any button to quit: ";
+			//Will ask the user if they want to stop the loop
+			cin>>msg_prompt;
+		}
```

Comment:
use a while loop to keep on asking the user for input if value not in the range of 1 to 10.

while(num > 1 || num > 10)
{
   cout<<"Invalid value, try again";
   cin>>num;
}